pkg/crypto/pedersen/points.go

```diff
@@ -2049,5 +2049,5 @@ func init() {
 		y, _ := new(big.Int).SetString(p[1], 16)
 		points[i] = point{x, y}
 	}
-	curve = weierstrass.Stark()
+	p = weierstrass.Stark().Params().P
```

Comment:
Let's go one step further and "hardcode" the initialisation of P instead of deriving it from the curve.